### PR TITLE
[Crossdata-21][Elastic Search Connector] Import & inventory

### PR DESCRIPTION
**Pull request by luismcl**
Import existing tables from Elastic Search

### CODE REVIEW

File: elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import org.elasticsearch.common.settings.ImmutableSettings
+import org.elasticsearch.hadoop.cfg.ConfigurationOptions._
+
+import scala.collection.mutable
+
+object ElasticSearchConnectionUtils {
+
+
+  def buildClient(parameters: Map[String, String]): ElasticClient = {
+    val host: String = parameters.getOrElse(ES_NODES, ES_NODES_DEFAULT)
```
Comment (darroyocazorla) [inline elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala]: ES_NODES can be a list of nodes

File: elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    val port: Int = parameters.getOrElse(ElasticNativePort, 9300).toString.toInt
```
Comment (darroyocazorla) [inline elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala]: parameters.getOrElse(ElasticNativePort, "9300").toInt

File: elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    val clusterName: String = parameters.get(ElasticCluster).get
```
Comment (darroyocazorla) [inline elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchConnectionUtils.scala]: val clusterName = parameters(ElasticCluster)

**Comment by luismcl** [comment]
download

**Comment by pmadrigal** [comment]
:+1:

**Comment by darroyocazorla** [comment]
+1

**Pull request merged**